core/config_utils_test.go

```diff
@@ -226,25 +227,75 @@ func TestGetServicePropertiesFromVCAP(t *testing.T) {
 	clearTestVCAP()
 }
 
-func TestLoadFromVCAPServices(t *testing.T) {
-	vcapServices := `{
-		"watson": [{
-			"credentials": {
-				"url": "https://gateway.watsonplatform.net/compare-comply/api",
-				"username": "bogus username",
-				"password": "bogus password",
-				"apikey": "bogus apikey"
-			}
-		}]
-	}`
-	os.Setenv("VCAP_SERVICES", vcapServices)
-	credential1 := loadFromVCAPServices("watson")
-	assert.Equal(t, "bogus apikey", credential1.APIKey)
+func TestLoadFromVCAPServicesWithServiceEntries(t *testing.T) {
+	data, err := ioutil.ReadFile("../resources/vcap_services.json")
+	isNil := assert.Nil(t, err, "There should be no i/o error.")
+	if !isNil {
```

Comment:
And one more nitpicky comment :)
I'm not a big fan of using messages in the assert statements, mainly because if a test failure occurs we're probably going to focus more on the stacktrace and line numbers in the test output to determine where things went wrong.
To some degree, the assert messages just add clutter to the testcase IMO.
I'm ok with you leaving them in there, it's just that I personally don't bother using them because I'm not convinced they add much/any value.

Reply:
There helpful in tests with multiple checks.